be/src/vec/exec/volap_scan_node.cpp

```diff
@@ -898,62 +765,23 @@ Status VOlapScanNode::build_key_ranges_and_filters() {
 Status VOlapScanNode::start_scan(RuntimeState* state) {
     RETURN_IF_CANCELLED(state);
 
-    VLOG_CRITICAL << "Eval Const Conjuncts";
```

Comment:
Is it good idea to remove those comment?

Reply:
I move this evaluation into `_normalize_predicate` so delete this comment.